db/structure.sql

```diff
@@ -2693,7 +2693,7 @@ CREATE TABLE public.supporters (
     deleted boolean DEFAULT false,
     organization character varying(255),
     imported_at timestamp without time zone,
-    country character varying(255) DEFAULT 'United States'::character varying,
+    country character varying(255),
```

Comment:
Why was this change made? I don't think it's a bad idea but I want to make sure I understand it.

Reply:
Because when a supporter was being created with an empty address, it was creating the primary address automatically because the country field had a default value. Since the supporter address table already has a default value for country, I figured I could remove this default to avoid that problem!